exercises/java/Makefile

```diff
@@ -0,0 +1,9 @@
+bootstrap:
+	wget https://d3pxv6yz143wms.cloudfront.net/11.0.5.10.1/java-11-amazon-corretto-devel-11.0.5.10-1.x86_64.rpm -O /tmp/java-corretto.rpm
```

Comment:
can't this just be yum installed?

https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/amazon-linux-install.html

Reply:
It wasn't in my repos when I searched for it (which surprised me), but maybe I fat fingered something.

Yep, looks like a fat finger. I'll fix.